source/unified-test-format/unified-test-format.rst

```diff
@@ -788,7 +790,7 @@ The structure of this object is as follows:
 - ``events``: Required array of one or more strings, which denote the events to
   be collected. Currently, only the following
   `CMAP <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`__
-  and `command monitoring <../command-monitoring/command-monitoring.rst>`__
```

Comment:
I'd leave this text as just "command monitoring" too since log messages are not covered by this field.